example/arcfullsim.py

```diff
@@ -104,54 +104,38 @@ def setupCerenkov(kernel):
         outImagePrefix = "arcsim_"
         ROOT.gROOT.SetBatch(1)
         rootfile = ROOT.TFile(SIM.outputFile)
-        if not "edm4hep" in SIM.outputFile:
-            EVENT = rootfile.Get("EVENT")
-            EVENT.Draw(
-                "ArcCollection.position.Z():ArcCollection.position.phi()",
-                "((ArcCollection.cellID>>5)&0x7)==0",
-            )
-            ROOT.gPad.SaveAs(outImagePrefix + "barrel_" + SIM.gun.particle + ".png")
-            EVENT.Draw(
-                "ArcCollection.position.Y():ArcCollection.position.X()",
-                "((ArcCollection.cellID>>5)&0x7)==1",
-            )
-            ROOT.gPad.SaveAs(outImagePrefix + "endcapZpos_" + SIM.gun.particle + ".png")
-            EVENT.Draw(
-                "ArcCollection.position.Y():ArcCollection.position.X()",
-                "((ArcCollection.cellID>>5)&0x7)==2",
-            )
-            ROOT.gPad.SaveAs(outImagePrefix + "endcapZneg_" + SIM.gun.particle + ".png")
-            EVENT.Draw(
-                "ArcCollection.position.Y():ArcCollection.position.X()",
-                "((ArcCollection.cellID>>5)&0x7)==2&&ArcCollection.position.Y()>0&&ArcCollection.position.X()>0",
-            )
-            ROOT.gPad.SaveAs(outImagePrefix + "endcapZneg_" + SIM.gun.particle + "zoom.png")
-        else:
-            EVENT = rootfile.Get("events")
-            EVENT.Draw(
-                "ArcCollection.position.z:atan(ArcCollection.position.y/ArcCollection.position.x)",
-                "((ArcCollection.cellID>>5)&0x7)==0&& ArcCollection.position.x>0",
-            )
-            ROOT.gPad.SaveAs(outImagePrefix + "barrel_" + SIM.gun.particle + ".png")
-            EVENT.Draw(
-                "ArcCollection.position.y:ArcCollection.position.x",
-                "((ArcCollection.cellID>>5)&0x7)==1",
-            )
-            ROOT.gPad.SaveAs(outImagePrefix + "endcapZpos_" + SIM.gun.particle + ".png")
-            EVENT.Draw(
-                "ArcCollection.position.y:ArcCollection.position.x",
-                "((ArcCollection.cellID>>5)&0x7)==2",
-            )
-            ROOT.gPad.SaveAs(outImagePrefix + "endcapZneg_" + SIM.gun.particle + ".png")
-            EVENT.Draw(
-                "ArcCollection.position.y:ArcCollection.position.x",
-                "((ArcCollection.cellID>>5)&0x7)==2&& ArcCollection.position.x>0&& ArcCollection.position.y>0",
-            )
-            ROOT.gPad.SaveAs(outImagePrefix + "endcapZneg_" + SIM.gun.particle + "zoom.png")
-
+        number_of_events = []
+        EVENT = rootfile.Get("EVENT")
```

Comment:
Does this work when writing an EDM4hep file now?

Reply:
I don't know, you tell us

Reply:
EDM4hep can be dropped from the example, unless someone else has something against